addons/mail/static/src/views/activity/activity_controller.js

```diff
@@ -74,7 +74,7 @@ export class ActivityController extends Component {
                 },
             },
             {
-                onClose: () => this.model.load(),
+                onClose: () => this.model.load({ "domain": this.model.originalDomain }),
             }
         );
     }
```

Comment:
Wouldn't it make sense to also have it on `onReloadData`, it feels like you would want to preserve the domain there too.

(at that point maybe the model itself should have a reload method that preserves the domain :shrug: )

Reply:
Done! Thanks :slightly_smiling_face: